Plugins/MatrixPlugin/MatrixForm.cs

```diff
@@ -0,0 +1,257 @@
+using Chummer;
+using Chummer.Backend.Equipment;
+using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace MatrixPlugin
+{
+    public partial class MatrixForm : Form
+    {
+        private readonly MatrixLogic logic;
```

Comment:
If you made this private member start with an underscore like the standard way of notating things in modern C++ and C#, you wouldn't have to muck around with the `this` keyword in the constructor. See my other comment about consistent naming.